### PR TITLE
Homework/week06

### PR DESCRIPTION
**Pull request by v605690**
File name - HiplatelicsTests.swift
Add unit testing:
4 function test

File name - UIHiplatelicsUITests.swift
Add UI testing:
3 function test

### CODE REVIEW

File: HiplatelicsTests/HiplatelicsTests.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  
+  func testStampSeller() {
+    let unstamp: String = "hiplatelics"
+      
+    XCTAssert(unstamp.contains("hip"))
+    XCTAssertFalse(unstamp.contains("Hiplatelics"))
+  }
+  
+  func testTotalAmount() {
+    let total = itemPrices.totalAmount
```
Comment (dokun1) [inline HiplatelicsTests/HiplatelicsTests.swift:42]: Very good job organizing your tests!

**Review by dokun1 — APPROVED** [review]
Nice work! You get a grade of "Meets expectations" for this assignment, due to the coverage amount on your unit tests.

**Comment by dokun1** [comment]
I do have some feedback on the layout of your app. The APIs allow you to do this, obviously, but it's _generally_ not a good idea to embed a TabBar in the root of a view that is displayed modally. TabBar's are good navigational components for the true root of an app, and should be difficult to navigate away from. Check this out for some more information: https://developer.apple.com/design/human-interface-guidelines/tab-bars#Best-practices

**Comment by v605690** [comment]
I will review the TabBar guidelines

**Pull request merged**